test/test_pyseir_end_to_end_test.py

```diff
@@ -0,0 +1,20 @@
+import os
+import json
+import numpy as np
+from pyseir import cli
+from pyseir.utils import get_run_artifact_path, RunArtifact
+import libs.datasets.can_model_output_schema as schema
+
+
+def test__pyseir_end_to_end():
```

Comment:
About 2 minutes I think.

Reply:
I would have added components, but it is hard without mocks since there is dependency structure.  And mocks take a long time, which I sadly do not have much of :(